CODEOWNERS

```diff
@@ -0,0 +1,2 @@
+* ataylor@directactioneverywhere.com
+* jake@directactioneverywhere.com
```

Comment:
is this email connected to your account or do you want me to use your github username or personal email? @jakehobbs

Reply:
that email is fine. but interesting, i didn't realize you can use emails as code owners. at work we use github usernames.